src/ui/public/share/index.html

```diff
@@ -0,0 +1,39 @@
+<form role="form" class="vis-share">
+  <p>
+    <input type="checkbox" ng-model="shortenUrls" id="shortenUrls" />
+    <label for="shortenUrls">Shorten URLs</label>
+  </p>
+  <p>
+    <div class="input-group">
+      <label>
+        Embed this {{objectType}}
+        <small>Add to your html source. Note all clients must still be able to access kibana</small>
+      </label>
+      <div class="form-control" disabled ng-if="!shortenUrls">
```

Comment:
It might be preferable to have these in `<textarea readonly="readonly"></textarea>` and have it so when you click inside, it auto-selects the block for easy copying. Kind of like is shown in this trick: https://css-tricks.com/snippets/javascript/auto-select-textarea-text/